target/linux/ramips/dts/mt7621_tplink_deco-x20-v3.dts

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "tplink,deco-x20-v3", "mediatek,mt7621-soc";
+	model = "TP-Link Deco X20 v3";
+
+	aliases {
+		label-mac-device = &gmac0;
+		led-boot = &led_system_blue;
+		led-failsafe = &led_system_blue;
+		led-running = &led_system_green;
+		led-upgrade = &led_system_red;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system_green: green {
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_system_blue: blue {
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_system_red: red {
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		led {
+			label = "led";
+			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_LIGHTS_TOGGLE>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	ubi-concat {
+		compatible = "mtd-concat";
+		devices = <&ubiconcat0>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "ubi";
+				reg = <0x0 0x2200000>;
+			};
+		};
+	};
+};
+
+&ethernet {
+	reg = <0x1e100000 0xe000>;
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+	compatible = "fixed-partitions";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+		partition@0 {
+			label = "uboot";
+			reg = <0x0 0x100000>;
+		};
+
+		partition@100000 {
+			label = "ubootenv";
+			reg = <0x100000 0x100000>;
+		};
+		
+		partition@200000 {
+			label = "radio";
+			reg = <0x200000 0x100000>;
+			read-only;
+		};
+		
+		partition@300000 {
+			label = "os0";
+			reg = <0x300000 0x2A00000>;
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "kernel";
+				reg = <0x0 0x800000>;
+			};
+
+			ubiconcat0: partition@800000 {
+				label = "ubi-os0";
+				reg = <0x800000 0x2200000>;
+			};
+		};
+		
+		partition@2d00000 {
+			label = "os1";
+			reg = <0x2d00000 0x2A00000>;
+		};
+		
+		partition@5700000 {
+			label = "factory_data";
+			reg = <0x5700000 0x900000>;
+		};
+		
+		partition@6000000 {
+			label = "runtime_data";
+			reg = <0x6000000 0x1100000>;
+		};
+		
+		ubiconcat2:partition@7100000 {
+			label = "balance_flash";
+			reg = <0x7100000 0xE80000>;
+		};
+	};
+};
+
+&state_default {
+	gpio {
+		groups = "uart2", "uart3";
+		function = "gpio";
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie1 {
+	status = "okay";
```

Comment:
```suggestion
```
The default status is already set as `"okay"`.

Reply:
Removed